retired/2022-2023/test_activation_shapes.py

```diff
@@ -1,7 +1,7 @@
 import matplotlib.pyplot as plt
 import numpy as np
 from enum import Enum
-
+from engine.grid import Grid
```

Comment:
I think we can delete this line and 164 since it wasn't used in the original file Aditya made (prob got added when we were testing stuff between this and the test file)